src/tool/hpcrun/ompt/ompt-device.c

```diff
@@ -537,9 +539,9 @@ ompt_device_load(int device_num,
 {
   PRINT("ompt_device_load->%s, %d\n", filename, device_num);
 
-#if 0 // FIXME
-  cupti_load_callback_cuda(module_id, host_addr, bytes);
-#endif
+  uint32_t loadmap_module_id;
+  bool mark_used = true;
```

Comment:
(Same as previous comment).
This constant does not need to be a variable. Using `true /* mark_used */` in the argument list is equivalently readable. (At the very least make it `const`).

Reply:
done